server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestGetIndicesAction.java

```diff
@@ -41,6 +46,14 @@
  */
 public class RestGetIndicesAction extends BaseRestHandler {
 
+    private static final DeprecationLogger deprecationLogger = new DeprecationLogger(LogManager.getLogger(RestGetIndicesAction.class));
+    static final String TYPES_DEPRECATION_MESSAGE = "[types removal] The response format of get indices requests will change in "
+            + "the next major version. Please start using the `include_type_names` parameter set to `false` in the request to "
```

Comment:
Small typo: `include_type_names` -> `include_type_name`.

Reply:
Good one, I was thinking about using the constant itself at some point to avoid typos like this but then decided against it for better readability. Will correct this.